docs/batch_inference_with_ts.md

```diff
@@ -106,7 +106,7 @@ $ curl -X POST "localhost:8081/models?url=resnet-152.mar&batch_size=8&max_batch_
 $ curl localhost:8081/models/resnet-152
 {
   "modelName": "resnet-152",
-  "modelUrl": "https://s3.amazonaws.com/model-server/model_archive_1.0/examples/resnet-152-batching/resnet-152.mar",
+  "modelUrl": "https://endpoint/model-server/model_archive_1.0/examples/resnet-152-batching/resnet-152.mar",
```

Comment:
This endpoint needs to be the real one from local store that was created in previous article.